sdks/java/core/src/main/java/org/apache/beam/sdk/options/PortablePipelineDebugOptions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.options;
+
+import com.google.auto.service.AutoService;
+import com.google.common.collect.ImmutableList;
+import org.apache.beam.sdk.annotations.Experimental;
+
+/** Pipeline options for debug flags for portable runner. */
+@Experimental
+@Hidden
+public interface PortablePipelineDebugOptions extends ExperimentalOptions, PipelineOptions {
```

Comment:
This option is more specific than portable jobs. It should rather be ManualDockerEnvironmentOptions, say, and not extend other options.